libs/shared/user-profile-card/src/lib/shared-user-profile-card.tsx

```diff
@@ -88,52 +77,25 @@ export const SharedUserProfileCard = (props: SharedUserProfileCardProps) => {
           </Typography>
         )}
         <CardActions>
-          <IconButton
-            aria-label="facebook"
-            href={props.facebook}
-            disabled={!props.facebook}
-            className="profile-card-social-button"
-          >
-            <FacebookIcon
-              className="profile-card-social-icon"
-              style={{ color: '#3b5999' }}
-            />
-          </IconButton>
-          <IconButton
-            aria-label="twitter"
-            href={props.twitter}
-            disabled={!props.twitter}
-            className="profile-card-social-button"
-          >
-            <TwitterIcon
-              className="profile-card-social-icon"
-              style={{ color: '#55acee' }}
-            />
-          </IconButton>
-          <IconButton
-            aria-label="linkedin"
-            href={props.linkedin}
-            disabled={!props.linkedin}
-            className="profile-card-social-button"
-          >
-            <LinkedInIcon
-              className="profile-card-social-icon"
-              style={{ color: '#55acee' }}
-            />
-          </IconButton>
-          <IconButton
-            aria-label="instagram"
-            href={props.instagram}
-            disabled={!props.instagram}
-            className="profile-card-social-button"
-          >
-            <InstagramIcon
-              className="profile-card-social-icon"
-              style={{ color: '#55acee' }}
-            />
-          </IconButton>
+          {props.socialLinks &&
+            props.socialLinks[0] &&
```

Comment:
remove this line, you checking length below

Reply:
by default, socialLinks = [
  null
]

so socialLinks.length = 1 

I want it to NOT work when there is null (database empty) by default, socialLinks[0] checks that